app/helpers/samples_helper.rb

```diff
@@ -55,7 +55,7 @@ def parsed_samples_for_s3_path(s3_path, project_id, host_genome_id)
                            host_genome_id: host_genome_id,
                            status: 'created' }
     s3_path.chomp!('/')
-    command = "aws s3 ls #{s3_path}/ | grep -v Undetermined"
+    command = "aws s3 ls #{s3_path}/ | grep -v Undetermined |grep fast"
```

Comment:
I notice missing space between `|` and `grep fast`.

Reply:
ok!